contracts/PriceFeedWithoutClearing.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.2;
+pragma experimental ABIEncoderV2;
+
+import './PriceVerifier.sol';
+import './IPriceFeed.sol';
+import 'hardhat/console.sol';
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract PriceFeedWithoutClearing is IPriceFeed, PriceModel, Ownable {
```

Comment:
How about calling it just PrriceFeed and extending by adding one method (+ one line to set current setter) to have the clearing version.

Reply:
Great idea, done